save-demo/src/main/kotlin/com/saveourtool/save/demo/service/DemoService.kt

```diff
@@ -104,4 +107,17 @@ class DemoService(
     ): Mono<Demo> = blockingToMono {
         findBySaveourtoolProject(organizationName, projectName)
     }.switchIfEmptyToNotFound(lazyMessage)
+
+    /**
+     * @param demo [Demo] entity
+     * @param version version of demo
+     * @return [Mono] of [Unit]
+     */
+    fun delete(demo: Demo, version: String): Mono<StringResponse> = stop(demo)
+        .let { dependencyStorage.list(demo, version) }
+        .concatMap { dependencyStorage.delete(it) }
+        .collectList()
+        .publishOn(Schedulers.boundedElastic())
```

Comment:
why not `blockingToMono`?

Reply:
Well, it is actually the same but without additional `Mono` creation

Reply:
yeap, but it's more visual to show blocking operations. but up to you